superduperdb/ext/transformers/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .model import Pipeline  # , TransformersTrainerConfiguration
+from .model import TextClassificationPipeline  # , TransformersTrainerConfiguration
```

Comment:
# ?

Reply:
Everything we do is specific to text classification. For the other hugging face tasks, we would need custom solutions.